source/sympp/functions/mathematics.h

```diff
@@ -22,9 +22,10 @@ namespace sympp {
 
     sym csc(const sym &);
 
-    sym sinh(const sym &);
+   // This two functions was definided in specific file
+   // sym sinh(const sym &);
 
-    sym cosh(const sym &);
```

Comment:
Não precisa remover. Basta usar `sympp::sinh` quando quiser diferenciar. Mesmo se estiver com `using namespace std`, não deveria dar conflito pois é outro overload. Se quiser, pode definir também uma outra versão (algo como `Sinh`) só dando forward para o `sympp::sinh` quando precisar. Só deveria precisar disso se 1) o compilar força `using namespace std` por algum erro no stdlib e 2) o compilar tem algum problema para identificar que isso é um overload diferente. É uma situação bem específica.

Reply:
Ola,

Utilizando sym coshD = sym(sympp::cosh(D)); e com as definições de cosh e sinh no mathematics.h, o erro apresentado é este abaixo: 

![Seleção_455](https://user-images.githubusercontent.com/57972585/96169861-2ed09800-0ef9-11eb-98e3-62a16fc2dfa8.png)

Ao alterar as versões para Sinh e Cosh fica resolvido.

[]s